docs/examples/table/main.js

```diff
@@ -0,0 +1,71 @@
+// @flow strict
+import { type Node } from 'react';
+import { Table, Flex, Text } from 'gestalt';
+
+export default function Example(): Node {
+  return (
+    <Flex alignItems="center" gap={4} height="100%" justifyContent="center" width="100%">
+      <Table accessibilityLabel="Basic Table">
+        <Table.Header>
+          <Table.Row>
+            <Table.HeaderCell>
```

Comment:
consider DRYing out Table code to better represent how data is likely to be used by our consumers

Reply:
DONE